package.json

```diff
@@ -31,6 +32,7 @@
     "zip": "^1.2.0"
   },
   "devDependencies": {
+    "lodash-cli": "^4.17.4",
```

Comment:
Is this how we did this in the php plugin? I think the built lib should not be checked in to git, but should be published to npmjs. This requires another key in package.json?

Reply:
in the php plugin, the dist folder is included
not sure what to do here